packages/api/endpoints/rules.js

```diff
@@ -16,10 +14,11 @@ const { RecordDoesNotExist } = require('../lib/errors');
  * @return {undefined}
  */
 function list(event, cb) {
-  const search = new Search(event, 'rule');
-  search.query().then(response => cb(null, response)).catch((e) => {
-    cb(e);
-  });
+  const rules = new models.Rule();
+  return rules
+    .scan()
```

Comment:
We should continue to use the ElasticSearch for returning results for the list endpoint, otherwise, none of our query parameters will work.